----------------------------------------------------------------
Fri Oct 11 12:34:31 CEST 2019:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.14.2.0 - (1828579): instance a816c00e-016d-ba62-5b05-0000061d3368 
on database directory C:\Users\alumno\Desktop\ESCUELA_JAVA\escuela_java.git\08_Spring\com.sinensia.springrestusersdemo\my_db_tmp with class loader sun.misc.Launcher$AppClassLoader@764c12b6 
Loaded from file:/C:/Users/alumno/.m2/repository/org/apache/derby/derby/10.14.2.0/derby-10.14.2.0.jar
java.vendor=Oracle Corporation
java.runtime.version=1.8.0_221-b11
user.dir=C:\Users\alumno\Desktop\ESCUELA_JAVA\escuela_java.git\08_Spring\com.sinensia.springrestusersdemo
os.name=Windows 7
os.arch=amd64
os.version=6.1
derby.system.home=null
Database Class Loader started - derby.database.classpath=''
----------------------------------------------------------------
Fri Oct 11 12:36:32 CEST 2019:
Shutting down instance a816c00e-016d-ba62-5b05-0000061d3368 on database directory C:\Users\alumno\Desktop\ESCUELA_JAVA\escuela_java.git\08_Spring\com.sinensia.springrestusersdemo\my_db_tmp with class loader sun.misc.Launcher$AppClassLoader@764c12b6 
Fri Oct 11 12:36:33 CEST 2019 Thread[Thread-23,5,main] Cleanup action starting
ERROR XBM0I: Directory my_db_tmp cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
Fri Oct 11 12:36:33 CEST 2019 Thread[Thread-23,5,main] Cleanup action starting
ERROR XBM0I: Directory my_db_tmp cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
----------------------------------------------------------------
Fri Oct 11 12:36:34 CEST 2019:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.14.2.0 - (1828579): instance 81260031-016d-ba62-5b05-0000061d3368 
on database directory C:\Users\alumno\Desktop\ESCUELA_JAVA\escuela_java.git\08_Spring\com.sinensia.springrestusersdemo\my_db_tmp with class loader sun.misc.Launcher$AppClassLoader@764c12b6 
Loaded from file:/C:/Users/alumno/.m2/repository/org/apache/derby/derby/10.14.2.0/derby-10.14.2.0.jar
java.vendor=Oracle Corporation
java.runtime.version=1.8.0_221-b11
user.dir=C:\Users\alumno\Desktop\ESCUELA_JAVA\escuela_java.git\08_Spring\com.sinensia.springrestusersdemo
os.name=Windows 7
os.arch=amd64
os.version=6.1
derby.system.home=null
Database Class Loader started - derby.database.classpath=''
----------------------------------------------------------------
Fri Oct 11 12:38:08 CEST 2019:
Shutting down instance 81260031-016d-ba62-5b05-0000061d3368 on database directory C:\Users\alumno\Desktop\ESCUELA_JAVA\escuela_java.git\08_Spring\com.sinensia.springrestusersdemo\my_db_tmp with class loader sun.misc.Launcher$AppClassLoader@764c12b6 
Fri Oct 11 12:38:08 CEST 2019 Thread[Thread-40,5,main] Cleanup action starting
ERROR XBM0I: Directory my_db_tmp cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
Fri Oct 11 12:38:08 CEST 2019 Thread[Thread-40,5,main] Cleanup action starting
ERROR XBM0I: Directory my_db_tmp cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
----------------------------------------------------------------
Fri Oct 11 12:38:10 CEST 2019:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.14.2.0 - (1828579): instance e3a74057-016d-ba62-5b05-0000061d3368 
on database directory C:\Users\alumno\Desktop\ESCUELA_JAVA\escuela_java.git\08_Spring\com.sinensia.springrestusersdemo\my_db_tmp with class loader sun.misc.Launcher$AppClassLoader@764c12b6 
Loaded from file:/C:/Users/alumno/.m2/repository/org/apache/derby/derby/10.14.2.0/derby-10.14.2.0.jar
java.vendor=Oracle Corporation
java.runtime.version=1.8.0_221-b11
user.dir=C:\Users\alumno\Desktop\ESCUELA_JAVA\escuela_java.git\08_Spring\com.sinensia.springrestusersdemo
os.name=Windows 7
os.arch=amd64
os.version=6.1
derby.system.home=null
Database Class Loader started - derby.database.classpath=''
----------------------------------------------------------------
Fri Oct 11 12:38:13 CEST 2019:
Shutting down instance e3a74057-016d-ba62-5b05-0000061d3368 on database directory C:\Users\alumno\Desktop\ESCUELA_JAVA\escuela_java.git\08_Spring\com.sinensia.springrestusersdemo\my_db_tmp with class loader sun.misc.Launcher$AppClassLoader@764c12b6 
Fri Oct 11 12:38:13 CEST 2019 Thread[Thread-57,5,main] Cleanup action starting
ERROR XBM0I: Directory my_db_tmp cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
Fri Oct 11 12:38:13 CEST 2019 Thread[Thread-57,5,main] Cleanup action starting
ERROR XBM0I: Directory my_db_tmp cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
----------------------------------------------------------------
Fri Oct 11 12:38:14 CEST 2019:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.14.2.0 - (1828579): instance 44ee8065-016d-ba62-5b05-0000061d3368 
on database directory C:\Users\alumno\Desktop\ESCUELA_JAVA\escuela_java.git\08_Spring\com.sinensia.springrestusersdemo\my_db_tmp with class loader sun.misc.Launcher$AppClassLoader@764c12b6 
Loaded from file:/C:/Users/alumno/.m2/repository/org/apache/derby/derby/10.14.2.0/derby-10.14.2.0.jar
java.vendor=Oracle Corporation
java.runtime.version=1.8.0_221-b11
user.dir=C:\Users\alumno\Desktop\ESCUELA_JAVA\escuela_java.git\08_Spring\com.sinensia.springrestusersdemo
os.name=Windows 7
os.arch=amd64
os.version=6.1
derby.system.home=null
Database Class Loader started - derby.database.classpath=''
----------------------------------------------------------------
Fri Oct 11 12:41:51 CEST 2019:
Shutting down instance 44ee8065-016d-ba62-5b05-0000061d3368 on database directory C:\Users\alumno\Desktop\ESCUELA_JAVA\escuela_java.git\08_Spring\com.sinensia.springrestusersdemo\my_db_tmp with class loader sun.misc.Launcher$AppClassLoader@764c12b6 
Fri Oct 11 12:41:51 CEST 2019 Thread[Thread-74,5,main] Cleanup action starting
ERROR XBM0I: Directory my_db_tmp cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
Fri Oct 11 12:41:51 CEST 2019 Thread[Thread-74,5,main] Cleanup action starting
ERROR XBM0I: Directory my_db_tmp cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
----------------------------------------------------------------
Fri Oct 11 12:41:52 CEST 2019:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.14.2.0 - (1828579): instance a666c073-016d-ba62-5b05-0000061d3368 
on database directory C:\Users\alumno\Desktop\ESCUELA_JAVA\escuela_java.git\08_Spring\com.sinensia.springrestusersdemo\my_db_tmp with class loader sun.misc.Launcher$AppClassLoader@764c12b6 
Loaded from file:/C:/Users/alumno/.m2/repository/org/apache/derby/derby/10.14.2.0/derby-10.14.2.0.jar
java.vendor=Oracle Corporation
java.runtime.version=1.8.0_221-b11
user.dir=C:\Users\alumno\Desktop\ESCUELA_JAVA\escuela_java.git\08_Spring\com.sinensia.springrestusersdemo
os.name=Windows 7
os.arch=amd64
os.version=6.1
derby.system.home=null
Database Class Loader started - derby.database.classpath=''
----------------------------------------------------------------
Fri Oct 11 12:48:53 CEST 2019:
Shutting down instance a666c073-016d-ba62-5b05-0000061d3368 on database directory C:\Users\alumno\Desktop\ESCUELA_JAVA\escuela_java.git\08_Spring\com.sinensia.springrestusersdemo\my_db_tmp with class loader sun.misc.Launcher$AppClassLoader@764c12b6 
Fri Oct 11 12:48:54 CEST 2019 Thread[Thread-91,5,RMI Runtime] Cleanup action starting
ERROR XBM0I: Directory my_db_tmp cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
Fri Oct 11 12:48:54 CEST 2019 Thread[Thread-91,5,RMI Runtime] Cleanup action starting
ERROR XBM0I: Directory my_db_tmp cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
